mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -255,6 +255,11 @@ const COMPATIBLE_BY_DEFAULT_TYPES = {
   dictionary: true
 };
 
+// A set of addon IDs which are allowed to run in safe mode.
+const gSafeModeWhitelistedAddons = new Set([
+  "cliqz@cliqz.com"
```

Comment:
we want to allow all `cliqz.com` addons. ATM there is only one `userassessment@cliqz.com`